include/boost/geometry/policies/compare.hpp

```diff
@@ -41,7 +42,7 @@ template
 <
     typename Point = void,
     int Dimension = -1,
-    typename CSTag = void
+    typename Strategy = void
```

Comment:
In summary, this is the main thing I'm worried about.

Reply:
See above.